mobile/src/components/uam-groups-create/uam-groups-create.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { FC, useEffect } from 'react';
+import { ScrollView, View } from 'react-native';
+
+import { AppScreenName, PaginationDefaultValue } from '~/common/enums/enums';
+import { GroupsCreateRequestDto } from '~/common/types/types';
+import { Button, Input, Text } from '~/components/common/common';
+import { CREATE_GROUP_DEFAULT_PAYLOAD } from '~/components/uam-groups-create/common/types/types';
+import {
+  GroupsTable,
+  UsersTable,
+} from '~/components/uam-groups-create/components/conponents';
+import {
+  useAppDispatch,
+  useAppForm,
+  useAppNavigate,
+  useAppSelector,
+  usePagination,
+  useSelectedItems,
+} from '~/hooks/hooks';
+import { getUsers } from '~/store/uam/actions';
+import { createGroup, getPermissions } from '~/store/uam-groups-create/actions';
+import { groupCreateClient } from '~/validation-schemas/validation-schemas';
+
+import { styles } from './styles';
+
+const UAMGroupsCreate: FC = () => {
+  const navigation = useAppNavigate();
+  const dispatch = useAppDispatch();
+  const { page: usersPage, handlePageChange } = usePagination();
+  const { control, handleSubmit, errors } = useAppForm<GroupsCreateRequestDto>({
+    defaultValues: CREATE_GROUP_DEFAULT_PAYLOAD,
+    validationSchema: groupCreateClient,
+  });
+
+  const { items: permissionIds, handleToggle: handleTogglePermissions } =
+    useSelectedItems<number>([]);
+
+  const { items: userIds, handleToggle: handleToggleUsers } =
+    useSelectedItems<number>([]);
+
+  const paginationForUsersTable = {
+    page: usersPage,
+    setPage: handlePageChange,
+  };
+
+  const { users, permissions } = useAppSelector(
+    (state) => state.uamGroupCreation,
+  );
+
+  const handleCreateGroup = (): void => {
+    dispatch(
+      createGroup({
+        name: control._formValues.name,
+        permissionIds,
+        userIds,
+      }),
+    );
+    navigation.navigate(AppScreenName.UAM);
+  };
+
+  const handleGetUsers = (page: number): void => {
+    dispatch(
+      getUsers({
+        page: page,
+        count: PaginationDefaultValue.DEFAULT_COUNT,
+      }),
+    );
+  };
+
+  useEffect(() => {
+    handleGetUsers(usersPage);
```

Comment:
I think we don\`t need `handleGetUsers`, it only used into one place.

Reply:
Maybe you are right!